app/models/user.rb

```diff
@@ -30,4 +31,36 @@ def full_name
   def proper_name
     [last_name, first_name].join ', '
   end
+
+  def self.import_from_xml(xml)
+    statuses = Hash.new(0)
```

Comment:
Right, this doesn't have the same issues as `Hash.new([])`, because `0 + 1` produces a new `Fixnum`, it doesn't modify the existing one in place.